Champions/Taric/W.cs

```diff
@@ -0,0 +1,80 @@
+using System.Linq;
+using GameServerCore;
+using GameServerCore.Domain;
+using GameServerCore.Domain.GameObjects;
+using GameServerCore.Enums;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+using LeagueSandbox.GameServer.GameObjects.AttackableUnits.AI;
+using LeagueSandbox.GameServer.GameObjects.Stats;

```

Comment:
There should be some rework to Stats using, as in its current state there are uses of lib.

Reply:
about Stats, what if I use "target.Stats.~~" to code? It will be same result that use from lib, or it would be OK

Reply:
Line 9 fix(with all connecting with codes) : Use "target.Stats.~~" instead of "StatModifier".